Added a test for finding variant annotation sets by their ID. fixed #144

david4096 opened the pull request:
Ref server is currently failing this test without the GET endpoint implemented.

sarahhunt commented on cts-java/src/test/java/org/ga4gh/cts/api/variantAnnotation/VariantAnnotationsSearchIT.java:
```diff
@@ -140,6 +140,23 @@ public void checkTranscriptEffects() throws AvroRemoteException {
     }
 
     /**
+     * Check to see if variant annotations can be gotten by their ID.
```
variant annotations => variant annotation sets

sarahhunt commented on cts-java/src/test/java/org/ga4gh/cts/api/variantAnnotation/VariantAnnotationsSearchIT.java:
```diff
@@ -140,6 +140,23 @@ public void checkTranscriptEffects() throws AvroRemoteException {
+        // Find a variant annotation set ID using search
+        final String variantAnnotationSetId = Utils.getVariantAnnotationSetId(client);
+
+        // Get a variant set using GET
```
variant => variant annotation set

sarahhunt commented:
Hi @david4096 
This one looks good as a test of the GET variantannotationsets/{id} endpoint. 
The comments are a little confusing (as highlighted) and it would be clearer to call it something like 'checkGetVariantAnnotationSetsById'. It may be better in a seperate file too (VariantAnnotationSetsGetByIdIT.java?) as the existing file names all suggest checks on /search endpoints. 

I could also see a point to an additional similar test in VariantAnnotationsSearchIT.java to check the consistency of the VariantAnnotation record. It would pull out a few variant annotation records, extract the variantAnnotationSetId string for each and do a GET look up to check it isn't garbage.

david4096 commented:
Hi @sarahhunt! The fetch by ID tests for all the other endpoints exist within the `*SearchIT` files for the other endpoints, so we can leave them for now. I moved the test to the appropriate file and fixed the comments, thanks!

sarahhunt commented:
That's clearer now!
